README.md

```diff
@@ -6,168 +6,171 @@
-  - [@Discussion Portal](https://groups.google.com/d/forum/codeuino-devel).
-  - [@Codeuino Wiki](wiki.codeuino.org).
+Once we are on our new branch we can make changes to the code which address the issue. When we have made the required changes that address a particular issue, we need to commit that code to our branch. We can use the “git status” command to view the changes since our last commit.
 
+```
+git status
+```
 
-## Sponsors/Partners
+We then use the “git add” command to stage the changes for the next commit. It's possible to add the files one by one by spcifying their relative path afterwards. For example if we want to add the corrections.js file in the same directory,
 
-Support this project by becoming a sponsor/collaborator/partner/contributors. 
-[[Become a sponsor](https://codeuino.org/)]
+```
+git add corrections.js
+```
 
-#### Current
+However if you want to stage all the files that have been changed, it's also possible with the follwing command.
 
-| <a href="https://www.womenwhocode.com/"><img src="https://secure.meetupstatic.com/photos/event/2/6/d/b/600_456849947.jpeg" width="200px" height="100px"></a> | <a href="https://www.digitalocean.com/"><img src="https://upload.wikimedia.org/wikipedia/commons/f/ff/DigitalOcean_logo.svg" width="200px" height="100px"></a> |
-|:-:|:-:|
+```
+git add .
+```
 
+Next we will commit our staged changes using the “git commit” command. In this case we can use the following example:
 
-## Licensing
+```
+git commit -m "Fix readme typo"
+```
 
-[![GitHub license](https://img.shields.io/github/license/codeuino/Social-Platform-Donut?logo=github)](https://github.com/codeuino/Social-Platform-Donut/blob/master/LICENSE)
+The -m option allows us to specify a message for our commit. It’s good practice to try and provide a succinct, but descriptive message for your commits. This helps a reviewer understand at a high level what was addressed in each commit.
 
-<b>Donut An Open Source Social Collaborating platform is a web Based framework Software,
-  
-   Copyright (C) < 2018 Codeuino Contributors >
-This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License V3 as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
+At this point we have made and committed out changes local to our development machine. Our final step is to push the changes to our fork of the allReady repository up on GitHub. We can do that using the “git push” command. We need to specify the name of the remote that we want to push to and the name of the branch we want to push up.
 
-This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details at OSI <https://opensource.org/licenses/GPL-3.0>
+```
+git push origin 223
```

Comment:
@AuraOfDivinity git remote -v will only show upstream as the origin is not added, I guess.
So you can also add that step git remote add origin URL